source/AMDSMainSGMScaler.cpp

```diff
@@ -16,7 +16,7 @@ int main(int argc, char *argv[])
 
 	bool startServer = AMDSMain_Common::parseArgument("AcquamanDataServerSGMScaler", args, &interfaceType, &port);
 	if (startServer) {
-		AMDSRunTimeSupport::setDebugLevel(1);
+		AMDSRunTimeSupport::setDebugLevel(AMDSRunTimeSupport::All);
```

Comment:
Okay.  @XuetaoLiu-CLS can you change both of these levels to error only?  Once you do that you should be able to merge it in.